packages/wdio-cli/src/constants.js

```diff
@@ -167,6 +167,22 @@ export const QUESTIONNAIRE = [{
     when: /* istanbul ignore next */ (answers) => {
         return answers.backend === 'In the cloud using Experitest' && answers.expEnvPort !== '80' && answers.expEnvPort !== '443'
     }
+}, {
+    type: 'input',
+    name: 'env_user',
+    message: 'Environment variable for username',
+    default: 'LT_USERNAME',
+    when: /* istanbul ignore next */ (answers) => {
+        return answers.backend.indexOf('LambdaTest') > -1 && answers.hostname.indexOf('lambdatest.com') > -1
```

Comment:
Shouldn't this be better an or? In case the user enters a custom endpoint url that also ends up with lambdatest.com?

```suggestion
    when: /* istanbul ignore next */ (answers) => (
        answers.backend.indexOf('LambdaTest') > -1 ||
        answers.hostname.indexOf('lambdatest.com') > -1
    )
```

Reply:
If we use `|| in place of && `  then for other service provider `host address` eg `hub.testingbot.com`  will also show `LT_USERNAME` & `LT_ACCESS_KEY` for username & access key default value

Reply:
Ah, I see now. Makes sense!